modules/apps/dynamic-data-mapping/dynamic-data-mapping-taglib/src/main/resources/META-INF/resources/html/start.jsp

```diff
@@ -75,8 +75,37 @@
 							String linkCssClass = "dropdown-item palette-item";
 
 							Locale curLocale = LocaleUtil.fromLanguageId(curLanguageId);
+
+							if (curLanguageId.equals(defaultLanguageId)) {
+								linkCssClass += " active";
+							}
+
+							String translationStatus = LanguageUtil.get(request, "untranslated");
+							String translationStatusCssClass = "warning";
+
+							if (ddmFormValues != null) {
+								Set<Locale> ddmFormValuesAvailableLocales = ddmFormValues.getAvailableLocales();
+
+								if (ddmFormValuesAvailableLocales.contains(curLocale)) {
+									translationStatus = LanguageUtil.get(request, "translated");
+									translationStatusCssClass = "success";
+								}
+							}
+
+							if (defaultLanguageId.equals(curLanguageId)) {
```

Comment:
Hi @georgel-pop-lr,
Could this condition be together with the condition on lines 79-81? The condition is the same, but I would like to know why you separated.

Reply:
Yes, it makes sense.